src/cfn/getAllStackEvents.ts

```diff
@@ -0,0 +1,35 @@
+import * as _ from 'lodash';
+import * as aws from 'aws-sdk'
+
+import def from '../default';
+import eventIsFromSubstack from './eventIsFromSubstack';
+
+export const getSubStacksFromEvents = (events: aws.CloudFormation.StackEvents): Set<string> => {
```

Comment:
It looks like this is not used outside of the module.
```suggestion
const getSubStacksFromEvents = (events: aws.CloudFormation.StackEvents): Set<string> => {
```